docs/en/Configuration-Settings.md

```diff
@@ -8,31 +8,75 @@ priority: 1
 * Table of Contents
 {:toc}
 
-There are two types of configuration parameters for Alluxio:
+This page explains the configuration system of Alluxio and also provides recommendation on how customize the configuration
+for Alluxio users in different contexts.
 
-1. [Configuration properties](#configuration-properties) are used to configure the runtime settings
-of Alluxio system, and
-2. [System environment properties](#system-environment-properties) control the Java VM options to
-run Alluxio as well as some basic very setting.
+# Configuration in Alluxio
 
-# Configuration properties
+Alluxio runtime respects two sources of configuration settings:
 
-On startup, Alluxio loads the default (and optionally a site specific) configuration properties file
-to set the configuration properties.
+1. [Environment variables](#Environment-variables). This is a convenient way for beginner users or shell scripts to set
+a few basic properties when running Alluxio.
+2. [Configuration properties](#configuration-properties). This provides a way to customize any
+[supported Alluxio configure properties](#appendix).
 
-1. The default values of configuration properties of Alluxio are defined in
-`alluxio-default.properties`. This file can be found in Alluxio source tree and is typically
-distributed with Alluxio binaries. We do not recommend beginner users to edit this file directly.
+## Environment variables
 
-2. Each site deployment and application client can also override the default property values via
-`alluxio-site.properties` file. Note that, this file **must be in the classpath** of the Java VM in
-which Alluxio is running. The easiest way is to copy the site properties template in directory
+There are a few basic but very frequently used Alluxio configuration properties that can be set via the
+following environment variables:
+
+<table class="table table-striped">
+<tr><th>Environment Variable</th><th>Meaning</th></tr>
+<tr><td>`$ALLUXIO_MASTER_HOSTNAME`</td><td>hostname of Alluxio master, default to localhost.</td></tr>
```

Comment:
saying "default to ..." is not grammatically correct

either say "defaults to ..." or "using ... as the default"